src/result.js

```diff
@@ -1305,14 +1305,14 @@ var Result = CreateClass({
 
                         var supplementalDamageInfo = [];
                         const supplementalInfo = supplemental.collectSkillInfo(skilldata.supplementalDamageArray, {remainHP: m.data[key].remainHP});
-                        if ("keys" in supplementalInfo && "values" in supplementalInfo && "total" in supplementalInfo) {
+                        if ("keys" in supplementalInfo && "values" in supplementalInfo && "total" in supplementalInfo && supplementalInfo.total > 0) {
```

Comment:
it's safe to remove those property checks, when empty object was given
supplementalInfo.keys, values is [] (empty array) map() will just ignore. total is default value 0.